src/ServiceControl.Audit/ServiceControl.Audit.csproj

```diff
@@ -48,6 +48,7 @@
     <PackageReference Include="System.Memory" Version="4.5.4" />
     <PackageReference Include="System.Buffers" Version="4.5.1" />
     <PackageReference Update="System.Runtime.CompilerServices.Unsafe" Version="4.7.1" />
+    <PackageReference Include="System.Text.Encodings.Web" Version="4.7.2" />
```

Comment:
If might make sense to move this to [here](https://github.com/Particular/ServiceControl/blob/06754557fe54623b74bdf4e647620f3c8eb5f53f/src/TransitiveDependencies.Build.props). The same for the Monitoring and Primary projects. They all reference this